### PR TITLE
Show shipping selector in confirmation screen.

### PR DESCRIPTION
**Pull request by flavioarfaria**
This change refers to the task of building the UI to show the list of available shipping methods:

This image is an actual screenshot of the implemented feature. The shipping methods are not being mocked, they are already being provided by Unity.

### CODE REVIEW

File: Assets/Shopify/Examples/Scripts/Panels/CartPanel.cs [hunk 1]
```diff
@@ -44,7 +44,8 @@ public class CartPanel : MonoBehaviour {
         private void Start() {
             BackToProductsButton.onClick.AddListener(() => OnReturnToProducts.Invoke());
             CheckoutButton.onClick.AddListener(() => {
-                _cart.CheckoutWithWebView(
+				_cart.CheckoutWithNativePay(
```
Comment (sleroux) [inline Assets/Shopify/Examples/Scripts/Panels/CartPanel.cs]: I would keep this out of this patch for now until we add native pay to the panel.

File: android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/utils/BigDecimalConverter.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+
+    /**
+     * Reads nullable a {@link BigDecimal} representation from a {@link JSONObject}.
+     *
+     * @param obj the {@code JSONObject} to extract the decimal from
+     * @param property the JSON property name to read the decimal from
+     * @return a {@code BigDecimal} representation of the JSON-encoded decimal number or
+     * {@code null} if such representatoin is absent from the JSON object
```
Comment (sleroux) [inline android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/utils/BigDecimalConverter.java]: typo: representatoin -> representation

File: scripts/generator/graphql_generator/csharp/SDK/ShippingMethod.cs.erb [hunk 3]
```diff
@@ -1,8 +1,8 @@
-#if UNITY_IOS
-namespace <%= namespace %>.SDK.iOS {
+namespace <%= namespace %>.SDK {
     using System.Collections.Generic;
     using System.Collections;
     using System;
+    using Shopify.Unity.MiniJSON;
```
Comment (sleroux) [inline scripts/generator/graphql_generator/csharp/SDK/ShippingMethod.cs.erb]: Replace `Shopify.Unity` with `<%= namespace %>`

File: Assets/Shopify/Examples/Scripts/Panels/CartPanel.cs (hunk 1 above)
Comment (flavioarfaria) [reply Assets/Shopify/Examples/Scripts/Panels/CartPanel.cs]: https://github.com/Shopify/unity-buy-sdk/pull/346/commits/e8984029244952e3107f7db6b5d35bcc65dd5906

File: android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/utils/BigDecimalConverter.java (hunk 2 above)
Comment (flavioarfaria) [reply android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/utils/BigDecimalConverter.java]: https://github.com/Shopify/unity-buy-sdk/pull/346/commits/097dcb2ff180bde8216ea1f844b4a40cf3fd042f

File: scripts/generator/graphql_generator/csharp/SDK/ShippingMethod.cs.erb (hunk 3 above)
Comment (flavioarfaria) [reply scripts/generator/graphql_generator/csharp/SDK/ShippingMethod.cs.erb]: https://github.com/Shopify/unity-buy-sdk/pull/346/commits/6e3fc907800772302df2f46cdf600abe284cef5d

**Comment by flavioarfaria** [comment]
@sleroux thanks for the thorough review. Checkstyle has been [fixed](https://github.com/Shopify/unity-buy-sdk/pull/346/commits/891998e1808853d7ef6e9046e668d19b9717b119). Please never hesitate to point out any spelling or grammar mistakes in the docs, or whatever you think it might sound more natural. I'll be glad to make it as correct as possible.

**Comment by sleroux** [comment]
I think if you rebase from beta it should resolve the remaining issues on Circle CI.

**Pull request merged**